src/org/openstreetmap/josm/tools/Utils.java

```diff
@@ -1642,11 +1642,7 @@ public static double clamp(double val, double min, double max) {
             throw new IllegalArgumentException(MessageFormat.format("Parameter min ({0}) cannot be greater than max ({1})", min, max));
         } else if (val < min) {
             return min;
-        } else if (val > max) {
-            return max;
-        } else {
-            return val;
-        }
+        } else return Math.min(val, max);
```

Comment:
I think this is another `Math.clamp` location. Which we still cannot use. For now, we can probably just do `Math.max(min, Math.min(val, max))`.

Reply:
Yes, this is the clamp() function. I'll revert the changes as anyway when the switch happens to newer Java versions, this function can be deprecated and eventually removed.

Reply:
Reverted in 8eed35f8.